websocket/src/websocket.cpp

```diff
@@ -346,9 +351,17 @@ static int LuaSend(lua_State* L)
 
     wslay_event_queue_msg(conn->m_Ctx, &msg); // it makes a copy of the data
 #else
-
-    dmSocket::Result sr = Send(conn, string, string_length, 0);
-    if (dmSocket::RESULT_OK != sr)
+    EMSCRIPTEN_RESULT result;
+    int write_mode = dmScript::CheckTableNumber(L, 3, "type", DATA_TYPE_BINARY);
```

Comment:
Same as line #345. Can move out of #ifdef statements

Reply:
The default value is different (DATA_TYPE_BINARY vs WSLAY_BINARY_FRAME). They represent the same thing though.

Reply:
Yes, you can use DATA_TYPE_BINARY